cranelift/src/disasm.rs

```diff
@@ -150,10 +150,16 @@ cfg_if! {
                             .build()
                     }
                 }
-                Architecture::Aarch64 {..} => Capstone::new()
-                    .arm64()
-                    .mode(arch::arm64::ArchMode::Arm)
-                    .build(),
+                Architecture::Aarch64 {..} => {
+                    let mut cs = Capstone::new()
+                        .arm64()
+                        .mode(arch::arm64::ArchMode::Arm)
+                        .build()
+                        .map_err(|err| err.to_string())?;
+                    // Inline constants should be skipped
```

Comment:
This is fine, but maybe expand the comment a bit to explain (AArch64 backend uses inline constants rather than a separate constant pool for now; we don't want to show these in the disassembly).

Reply:
> we don't want to show these in the disassembly

This actually makes them show up as `.byte` directives. Before the function was truncated to just before the inline constant. I expanded the comment.